spec/models/report_spec.rb

```diff
@@ -520,11 +520,12 @@
 
       before do
         freeze_time
-
-        PostActionCreator.new(flagger, post, PostActionType.types[:spam], message: 'bad').perform
       end
 
       it "returns a report with data" do
+        result = PostActionCreator.new(flagger, post, PostActionType.types[:spam], message: 'bad').perform
```

Comment:
What was happening here is that the post action failed to create because the message used the word 'bad' which is a blocked watched word created in https://github.com/discourse/discourse/blob/8e9164fb60873e65b96e836bc3639f38b384e0b1/spec/models/user_profile_spec.rb#L8. While the `WatchedWord` does not exist when this test is run, the cache in `WordWatcher` still contains the word.